modules/administration/pages/maintenance-window.adoc

```diff
@@ -13,8 +13,70 @@ You will need to stop the spacewalk, SAP, and database services, along with any
 . Apply the latest updates.
 See xref:upgrade:server-upgrade.adoc[].
 . Update to the latest service pack, if required.
-See xref:upgrade:sp-migration.adoc[].
+See xref:upgrade:migrate-4x-4x.adoc[].
 . Reboot the server.
 . Check if the configuration is still correct.
 . On Salt systems, apply the highstate.
 . Start any stopped services.
+
+
+
+== Keep the Server Patched
```

Comment:
This section needs a fair bit of editing. Might be easier for me to take the branch? LMK.

Reply:
@Loquacity Yes, please take it!

Reply:
OK, I've moved the content in this section around a bit. Let me know what you think.

Reply:
I think thus far you did not push it ;)

Reply:
Check https://github.com/SUSE/doc-susemanager/pull/704/commits/7d63e0d8c1963ad042879f8ce4eb6028a2981d34

Reply:
Argl, I should have seen it ;)